### ISSUE TITLE
Releases page redirects to 404

### ISSUE THREAD

**Issue by capsulecorplab** [DocOps/asciidocsy-jekyll-theme#81]
https://asciidocsy.netlify.app/releases redirects to 404 page

**Comment by briandominick** [comment]
I don't think this URL (`/releases`) is supposed to be linked to from anywhere. How did you get that URL?

The history page is at https://asciidocsy.netlify.app/docs/theme/history, But I know you already saw that.

I'm going to try to get some more stuff squared away today. Definitely tell me if you find any other bugs Sean.

**Comment by capsulecorplab** [comment]
Top right corner of https://asciidocsy.netlify.app/, next to **About** is **Releases**

**Comment by briandominick** [comment]
Gah! You're right! Will fix.
